clusterloader2/README.md

```diff
@@ -48,12 +53,23 @@ Two always available parameters are ```{{.Name}}``` and ```{{.Index}}```
 which specifies object name and object replica index respectively. \
 Example of a template can be found here: [load rc template].
 
+### Overrides
+
+Overrides allow to inject new variables values to the template. \
+Many tests define input parameters. Input parameter is a variable
+that potentially will be provided by the test framework. Cause input parameters are optional,
+each reference should be opaqued with ```DefaultParam``` function that will
```

Comment:
Each reference has to be opaqued with ..., because input parameters are option and we need to handle cases when env variable is unset.

Reply:
OK. Fixed.